src/components/pre-playback-play-overlay-wrapper/pre-playback-play-overlay-wrapper.tsx

```diff
@@ -48,42 +51,59 @@ interface PrePlaybackPlayOverlayWrapperProps {
-  withText({
-    next: 'playlist.next',
-    startOver: 'controls.startOver'
-  })(
-    connect(mapStateToProps)(
-      ({isPlaybackEnded, sizeBreakpoint, relatedManager, next, startOver, eventManager, eventContext}: PrePlaybackPlayOverlayWrapperProps) => {
-        const [isAutoContinueCancelled, setIsAutoContinueCancelled] = useState(relatedManager.isAutoContinueCancelled);
+  withPlayer(
+    withText({
+      next: 'playlist.next',
+      startOver: 'controls.startOver',
+      title: 'controls.title'
+    })(
+      connect(mapStateToProps)(
+        ({
+          isPlaybackEnded,
+          sizeBreakpoint,
+          relatedManager,
+          next,
+          startOver,
+          title,
+          eventManager,
+          player,
+          eventContext
+        }: PrePlaybackPlayOverlayWrapperProps) => {
+          const [isAutoContinueCancelled, setIsAutoContinueCancelled] = useState(relatedManager.isAutoContinueCancelled);
 
-        useEffect(() => {
-          eventManager.listen(eventContext, RelatedInternalEvent.AUTO_CONTINUE_CANCELLED_CHANGED, ({payload}: {payload: boolean}) => {
-            setIsAutoContinueCancelled(payload);
-          });
-        }, []);
+          useEffect(() => {
+            eventManager.listen(eventContext, RelatedInternalEvent.AUTO_CONTINUE_CANCELLED_CHANGED, ({payload}: {payload: boolean}) => {
+              setIsAutoContinueCancelled(payload);
+            });
+          }, []);
 
-        if (!isPlaybackEnded) return <PrePlaybackPlayOverlay />;
-        else if (isAutoContinueCancelled && (sizeBreakpoint === PLAYER_SIZE.SMALL || sizeBreakpoint === PLAYER_SIZE.EXTRA_SMALL)) {
-          return (
-            <div className={styles.minimalPrePlaybackPlayOverlay}>
-              <div className={styles.buttonContainer}>
-                <Tooltip label={startOver}>
-                  <button tabIndex={0} aria-label={startOver} className={styles.prePlaybackPlayButton} onClick={() => relatedManager.startOver()}>
-                    <Icon type={IconType.StartOver} />
-                  </button>
-                </Tooltip>
-                <Tooltip label={next}>
-                  <button tabIndex={0} aria-label={next} className={styles.prePlaybackPlayButton} onClick={() => relatedManager.playNext()}>
-                    <Icon type={IconType.Next} />
-                  </button>
-                </Tooltip>
-                <div />
+          if (!isPlaybackEnded) return <PrePlaybackPlayOverlay />;
+          else if (isAutoContinueCancelled && (sizeBreakpoint === PLAYER_SIZE.SMALL || sizeBreakpoint === PLAYER_SIZE.EXTRA_SMALL)) {
+            return (
+              <div className={styles.minimalPrePlaybackPlayOverlay}>
+                <div className={styles.buttonContainer}>
+                  <Tooltip label={startOver}>
+                    <button
+                      tabIndex={0}
+                      aria-label={`${startOver}, ${title}: ${player.sources.metadata?.name}`}
```

Comment:
what if name is undefined ?

Reply:
name shouldn't be undefined. it's not possible to create entry without name